### PR TITLE
Bug fix to correctly handle the logic for 'maxEntries' Issue #2050

### PR DESCRIPTION
**Pull request by chamakura**
* Does the pull request solve a **related** issue? Yes
* If so, can you reference the issue? https://github.com/MichMich/MagicMirror/issues/2050
* What does the pull request accomplish? Use a list if needed.
** Removes the logic of filtering events by `maxEntries` from calendarfetcher.js
** Results in the logic of filtering to solely be done in calendar.js
* If it includes major visual changes please add screenshots.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ❤️ **Donate:** Enjoying MagicMirror²? [Please consider a donation!](https://magicmirror.builders/donate) With your help we can continue to improve the MagicMirror²
 
+## [20.11.1] - 2020-06-20
```
Comment (rejas) [inline CHANGELOG.md:8]: I guess you will have let @MichMich to decide if a bugfix release will happen for that...

Comment (chamakura) [reply CHANGELOG.md:8]: Ok. I only added this because the guidelines suggested me to edit the changelog. I'm happy to revert this change if needed.

**Comment by MichMich** [comment]
Please add the changelog to the 2.12 entry. Thanks.

**Comment by chamakura** [comment]
@MichMich: I updated the version to 2.12.0 - is that what you're suggesting?

**Comment by rejas** [comment]
Have you started your work from the master or the develop branch? Seems like the former which leads to these merge conflicts.

**Comment by chamakura** [comment]
Ah, sorry about that - I had started from master. Should I just abort this pull request and submit a new one?
Again, sorry for the back and forth here.

**Pull request closed without merging**